extension/.eslintrc.json

```diff
@@ -19,7 +19,13 @@
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/no-floating-promises": "error",
     "no-unused-vars": "off",
-    "no-console": "warn"
+    "no-console": "warn",
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
```

Comment:
it looks more like a CRLF issue related to git for windows. 

1. what's the effective config without adding this block?
2. does "auto" means CRLF on Windows and LF on unix-like os?

Reply:
> it looks more like a CRLF issue related to git for windows.
> 
> 1. what's the effective config without adding this block?

if we didn't add this block, the default value is `LF`. See https://prettier.io/docs/en/options.html#end-of-line
> 2. does "auto" means CRLF on Windows and LF on unix-like os?

`auto` means maintaining the existing end of lines. It looks like the files in the repo are using `LF` and when windows users clone this repo (with git config default value `core.autocrlf == true`), the cloned files are transformed into `CRLF`. But in prettier, the default EOL is still `LF`, which causes this issue.